src/netsight/cloudstorage/upgrade.py

```diff
@@ -0,0 +1,16 @@
+from Products.CMFCore.utils import getToolByName
+
+
+PROFILE_ID = 'profile-netsight.cloudstorage:default'
+
+
+def upgrade_to_0002(context, logger=None):
+    if logger is None:
+        from logging import getLogger
+        logger = getLogger('esdrt.content.upgrades.0001_0002')
```

Comment:
This looks like a copy-paste error in the logger name. Should be netsight.cloudstorage.upgrades or similar